### ISSUE TITLE
WebSocket echo test slide

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#258]
```
Added a WebSocket echo test to the WebSocket slide (slide #11) along with some 
descriptive text.

I'm not confident my check-in succeeded since it was my first time. So I've 
attached the files here just in case.
```

Original issue reported on code.google.com by `robin.zi...@gtempaccount.com` on 3 Nov 2010 at 8:53

Attachments:
- [html5rocks-websocket.zip](https://storage.googleapis.com/google-code-attachments/html5rocks/issue-258/comment-0/html5rocks-websocket.zip)

**Comment by GoogleCodeExporter** [comment]
```
Note: I also fixed the title so it says "WebSocket" which is the correct name. 
This means issue 256 has been addressed:

http://code.google.com/p/html5rocks/issues/detail?id=256

```

Original comment by `robin.zi...@gtempaccount.com` on 3 Nov 2010 at 8:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 3 Nov 2010 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
I have attached the diff of html5.html. Also, a new sub-directory in 
slides.html5rocks/src is required, called "websocket". It which contains the 
following files (also attached):

- websocket.css
- websocket.js
- kaazing-logo_99x16.png
- tls-lock.png

```

Original comment by `robin.zi...@gtempaccount.com` on 7 Nov 2010 at 5:47

Attachments:
- [html5rocks-websocket.zip](https://storage.googleapis.com/google-code-attachments/html5rocks/issue-258/comment-3/html5rocks-websocket.zip)

**Comment by GoogleCodeExporter** [comment]
```
Updated attachment with a small fix. Ignore any attachment in the previous 
comment and use this one.
```

Original comment by `robin.zi...@gtempaccount.com` on 7 Nov 2010 at 5:56

Attachments:
- [html5rocks-websocket.zip](https://storage.googleapis.com/google-code-attachments/html5rocks/issue-258/comment-4/html5rocks-websocket.zip)

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 9a25f1c767.
```

Original comment by `ericbide...@html5rocks.com` on 7 Nov 2010 at 4:45
- Changed state: **Fixed**
